src/Components/Header/MobileNavigation.js

```diff
@@ -0,0 +1,98 @@
+import { 
+    StyledMobileNav, StyledLi, 
+    StyledInputForHamburguer, StyledUlForHamburguer, 
+    StyledSpanForHamburguer, StyledHamburguerContainer, 
+    StyledLinkForHamburguer, StyledFlexContainer,
+    StyledLogoImg, StyledLogoBrand } from "../../utils/StyledComponents/Styled";
+
+import Search from "./Search";
+import { MdOutlineShoppingCart } from "react-icons/md";
+
+const iconStyle = {
+    fontSize: "2em",
+    color: "#f5f0ec",
+    marginLeft: "1em",
+    cursor: "pointer"
+}
+const hamburguer = {
+    position: "relative",
+    top: "1.5vh",
+}
+
+const MobileNavigation = ({getSearchResult}) => {
+    return (
+        <>
+        <StyledMobileNav>
+            <StyledHamburguerContainer>
+            <StyledFlexContainer>
+                    <div style={hamburguer}>
```

Comment:
why don't you create a styled for this? just to have a common approach

Reply:
Done, thanks for the suggestion